src/sdk/Audio/AudioOutputFormat.ts

```diff
@@ -519,6 +521,28 @@ export class AudioOutputFormatImpl extends AudioStreamFormatImpl {
                     speechSynthesisOutputFormatString,
                     "raw-44100hz-16bit-mono-pcm",
                     true);
+            case "amr-wb-16000h":
+                return new AudioOutputFormatImpl(
+                    AudioFormatTag.AMR_WB,
+                    1,
+                    16000,
+                    3052,
+                    2,
+                    16,
+                    speechSynthesisOutputFormatString,
+                    speechSynthesisOutputFormatString,
```

Comment:
are you sure this second line is correct.
looks like cut and past error, when compared to the other formats that have an actual literal format sting in the second position.